src/libraries/System.Net.Http/tests/StressTests/HttpStress/NuGet.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!-- Add public nuget feed. -->
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Isn't this default? I don't mind to be explicit but I'm wondering if we need it in case we do not need custom feeds

Reply:
Not if run from runtime repo, ours explicitly removes nuget.org: https://github.com/dotnet/runtime/blob/main/NuGet.config
So it all works in docker, where it's isolated, but when you ran it locally from S.N.Http/tests/... it failed due to one dependency coming from the public feed.